readme.md

```diff
@@ -138,7 +138,7 @@ Please be aware that videos can be outdated. Always verify them with the docs.
 ### Dev dApps
 
 - [R3D4](https://r3d4.fr) - Faucet, ESDT tools, Stats
-- [Convertes for Elrond Blockchain](http://207.244.241.38/elrond-converters) - This page offers an easy to use pack of convertors necessary for interacting with Elrond Blockchain.
```

Comment:
Hey @SkyzoxRobin, could we add it as a separate entry? I would like to keep the one with credits for Bogdan.

By the way, could you leave a fixed version of the description in the old one and add a different one for the new one? That would be perfect. Thanks.